string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,18 @@
+/* exported numVowels */
+function numVowels(string) {
+  var totalVowels = 0;
+  var vowels = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'];
+  for (var char of string) {
```

Comment:
If you can explain how a `for ... of` loop works, then it's approved 👍 , otherwise, use a plain `for` loop.